### PR TITLE
Add StealthEX to the exchanges list

### PR DESCRIPTION
**Pull request by Vtasz**

### CODE REVIEW

**Review by merc1er — CHANGES_REQUESTED** [review]
Invalid JSON. Did you try to build the site? CD failed.

**Comment by Vtasz** [comment]
> Invalid JSON. Did you try to build the site? CD failed.

Yeah, wanted to create a pull request to add our service on the website. Did it similarly to Simpleswap, or so I thought. Unfortunatelt my github knowledge is rather shallow. I've asked our devs to assist and they should be able to do it properly

File: app/data/exchanges.json [hunk 1]
```diff
@@ -103,6 +103,9 @@
     {
       "name": "BitcoinVN",
       "link": "https://bitcoinvn.io/en"
-    }
+    },
+    {
+      "name": "StealthEX",
+      "link": "https://stealthex.io/"
```
Comment (danger89) [inline app/data/exchanges.json:109]: Missing `}` after line 109.

**Review by danger89 — CHANGES_REQUESTED** [review]

**Comment by Vtasz** [comment]
Had one of our devs, has opened a proper pull request! 
Thank you!

**Pull request closed without merging**